AC3.2Unit2FinalAssessment/AC3.2Unit2FinalAssessment/ColorTableViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ColorTableViewController.swift
+//  AC3.2Unit2FinalAssessment
+//
+//  Created by Annie Tung on 10/6/16.
+//  Copyright © 2016 Annie Tung. All rights reserved.
+//
+
+import UIKit
+
+class ColorTableViewController: UITableViewController {
+    
+    internal var crayonData: [Crayon]?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        var crayons: [Crayon] = []
+        for c in crayolaColors {
+            if let crayon = Crayon(fromDict: c) {
+                crayons.append(crayon)
+            }
+            crayonData = crayons
+    }
+}
+
+    
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return crayolaColors.count
+    }
+    
```

Comment:
You need to implement the (_:cellForRowAtIndexPath:) method for your tableview to work properly. This method is not optional.